solr/core/src/java/org/apache/solr/security/JWTIssuerConfig.java

```diff
@@ -68,6 +68,11 @@
   private WellKnownDiscoveryConfig wellKnownDiscoveryConfig;
   private String clientId;
   private String authorizationEndpoint;
+  
+  public static boolean ALLOW_OUTBOUND_HTTP = Boolean.parseBoolean(System.getProperty("solr.auth.jwt.allowOutboundHttp", "false"));
+  public static final String ALLOW_OUTBOUND_HTTP_ERR_MSG = "Outbound non SSL protected JWT authentication urls are not enabled, start your nodes with -Dsolr.auth.jwt.allowOutboundHttp=true.";
+
+
```

Comment:
Unneccessary spaces

Reply:
"Jan-Lint-Bot" ;-)    Where is my Solr Java Lint checker ;-)